app/build.gradle

```diff
@@ -68,7 +68,8 @@ dependencies {
 
     // todo: remove if not needed
     implementation 'androidx.paging:paging-runtime-ktx:2.1.2'
-    implementation 'androidx.swiperefreshlayout:swiperefreshlayout:1.1.0' // load and display small chunks of data at a time
+    implementation 'androidx.swiperefreshlayout:swiperefreshlayout:1.1.0'
```

Comment:
@enavhasson if in used, move the declaration to a different section (the first one in this case since this is androidx lib) - because all the remaining will be deleted in the end